clang/test/CodeGen/deeply-nested-expressions.cpp

```diff
@@ -0,0 +1,29 @@
+// RUN: %clang_cc1 %s -emit-llvm -o - -Wstack-exhausted
```

Comment:
Maybe add a mention that we cannot `-verify` the warning because it's platform-specific (Windows does not fire it), but we still want to make sure the compiler can handle the deeply nested expressions.

Reply:
Resolving since we decided to remove this test.